### ISSUE TITLE
errors

### ISSUE THREAD

**Issue by caw67** [firebug/pixel-perfect#49]
i have no button in firebug and the layis not scrolling with broser scrolling.
and if i deactivate the mozilla developer toolbar the layer is also close

**Comment by janodvarko** [comment]
@ibuck: is this already fixed?

Honza

**Comment by mbuckley** [comment]
@janodvarko Nope..mistake on close.

@caw67 can you post your FF version and OS please?

Thanks!

**Comment by janodvarko** [comment]
And Firebug version...

Honza

**Comment by caw67** [comment]
ff 36.0, windows7, firebug 2.0.8

**Comment by caw67** [comment]
and pixelperfect opens the mozilla inspector bar when i click the pixelperfectbutton

**Comment by janodvarko** [comment]
Yep, Pixel Perfect 2 isn't integrated with Firebug 2.0.8, it only supports Firebug.next (aka Firebug 3)
I guess we could detect that Firebug 2 is installed...

Honza

**Comment by caw67** [comment]
installed firebug next. these errros 
 have no button in firebug and the layout not scrolling with broser scrolling.

**Comment by speedo-spin** [comment]
+1 
not scrolling with browser scrolling.

**Comment by janodvarko** [comment]
> not scrolling with browser scrolling.

Fixed as part of issue  #18

> have no button in firebug and the layout not scrolling with broser scrolling.

Does it work for you if you use Firefox Nightly?
https://nightly.mozilla.org/

Honza

**Comment by caw67** [comment]
its not fixed, same as before
i dont use ff nightly.

**Comment by speedo-spin** [comment]
works in Firefox Nightly, but doesn't work in general firefox

**Comment by janodvarko** [comment]
> works in Firefox Nightly, but doesn't work in general firefox

yes, the scrolling problem (layer should scroll with the page content) seem to be fixed in Nightly only

What about the other problem "no button in Firebug"? This works for me in Nightly as well as Firefox Release channel...

Honza

**Comment by speedo-spin** [comment]
I have pixelperfect button in firefox panel, so I see no issues according button.
Janodvarko, maybe you will be intrested in suppoorting css-x-fire addon for jetBrains ide?
https://youtrack.jetbrains.com/issue/WEB-14355
This addon make possible to save changes in css files depending on your changes in firebug. Right on fly. Currently it doesn't work with firebug 3

**Comment by janodvarko** [comment]
> Janodvarko, maybe you will be intrested in suppoorting css-x-fire addon for jetBrains ide?
> https://youtrack.jetbrains.com/issue/WEB-14355

Sounds interesting, thanks for the link, I'll do some analyses.

Honza

**Comment by SebastianZ** [comment]
> > Janodvarko, maybe you will be intrested in suppoorting css-x-fire addon for jetBrains ide?
> > https://youtrack.jetbrains.com/issue/WEB-14355
> 
> Sounds interesting, thanks for the link, I'll do some analyses.

There are also [cssUpdater](https://getfirebug.com/wiki/index.php/Firebug_Extensions#cssUpdater) and [FireFile](https://getfirebug.com/wiki/index.php/Firebug_Extensions#FireFile) targetting the same feature. I don't know about their status, though.

Where's the place to discuss extension suggestions in more detail?

Sebastian

**Comment by janodvarko** [comment]
We've been discussing this on our regular meetings for a while and we have a plan that we are focusing on ATM.

> Where's the place to discuss extension suggestions in more detail?

You can file a new issue in Firebug.next issue list if you like.

Honza
